src/components/Header.jsx

```diff
@@ -1,8 +1,28 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
+import { setTheme } from '../redux/features/themeSlice';
 import styles from '../styles.scss';
+
 function Header() {
+  const theme = useSelector(state => state.theme.theme);
+  const dispatch = useDispatch();
+  function switchTheme(themeName) {
+    dispatch(setTheme(themeName));
+  }
+
+  useEffect(() => {
+    const backgroundColorHeader = `var(--background-color-${theme})`;
+    const backgroundColorMain = `var(--background-color-main-${theme})`;
+    document.body.style.setProperty('--background-color', backgroundColorHeader);
+    document.body.style.setProperty('--background-color-main', backgroundColorMain);
```

Comment:
are you only switching the background color?

Reply:
Yep, all text colors looks good in both themes

Reply:
I can switch other colors if it needs